Html/home.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../css/homeStyle.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cairo:wght@400;600;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/da6da1fff8.js" crossorigin="anonymous"></script>
+    <title>Home</title>
+</head>
+<body>
+    <img class="shape1" src="../assets/shape1.png">
+    <img class="shape2" src="../assets/shape2.png">
+    <img class="shape3" src="../assets/shape3.png">
+    <img class="shape4" src="../assets/shape4.png">
+    <img class="shape5" src="../assets/shape5.png">
```

Comment:
the folder called `Assets` not `assets`

![Screenshot from 2021-04-14 00-12-26](https://user-images.githubusercontent.com/47992412/114622027-21347900-9cb6-11eb-8f84-23d4fbe5e756.png)


change the folder name to `assets`

Reply:
I modified it